cfgov/v1/templates/cdnadmin/index.html

```diff
@@ -11,49 +12,52 @@
     <div class="nice-padding">
         <h2>Flush Akamai</h2>
 
-        {% if user_can_purge %}
+    {% if user_can_purge %}
         <div class="help-block help-info">
-            <p>Enter a URL to flush it from the cache, or leave it blank to
-            flush the entire site cache</p>
+            <p>
+                Enter a full URL (e.g., <code>https://www.consumerfinance.gov/about-us/</code>)
+                to flush it from the cache, or leave it blank to flush the entire site cache.
+            </p>
         </div>
         <form method="post" id="flush_site">
             {% csrf_token %}
             <ul class="fields">
                 <li>
-            <div class="field url_field url_input">
-            <div class="input">
-            {{ form.url }}
-            </div>
-            </div>
-            </li>
-        </ul>
-        <button type="submit" name="flush" class="button" form="flush_site" />
-        Flush cache 
-        </button>
+                    <div class="field url_field url_input">
+                        <div class="input">
+                        {{ form.url }}
+                        </div>
+                    </div>
+                </li>
+            </ul>
+            <button type="submit" name="flush" class="button">Flush cache</button>
         </form>
-        {% else %}
+    {% else %}
         <div class="help-block help-info">
-            <p>You do not have permission to submit purge requests, but you
-            are welcome to view the history</p>
+            <p>
+                You do not have permission to submit purge requests,
+                but you are welcome to view the history.
+            </p>
         </div>
-        {% endif %}
-        <h2> History </h2>
+    {% endif %}
+
+        <h2>History</h2>
 
         <table class="listing">
             <thead>
-            <tr>
-                <td> When </td>
-                <td> Who </td>
-                <td> Result </td>
-            </tr>
+                <tr>
+                    <th>When</th>
+                    <th>Who</th>
+                    <th>Result</td>
```

Comment:
Shouldn't the closing tag be `</th>` here?

Reply:
Good catch; thank you! Will fix.

Reply:
Amended the commit, rebased on master, and force pushed.